api/src/permissions/modules/fetch-allowed-field-map/fetch-allowed-field-maps.ts

```diff
@@ -0,0 +1,100 @@
+import type { Accountability, PermissionsAction } from '@directus/types';
+import { difference, intersection, uniq } from 'lodash-es';
+import { fetchPermissions } from '../../lib/fetch-permissions.js';
+import { fetchPolicies } from '../../lib/fetch-policies.js';
+import type { Context } from '../../types.js';
+import { useCache } from '../../cache.js';
+
+export type FieldMap = Record<string, string[]>;
+export type FieldMapsResult = { allowed: FieldMap; inconsistent: FieldMap };
+
+export interface FetchFieldMapsOptions {
+	accountability: Pick<Accountability, 'user' | 'role' | 'roles' | 'ip' | 'admin' | 'app'> | null;
+	action: PermissionsAction;
+	fieldMapTypes: ('allowed' | 'inconsistent')[];
```

Comment:
```suggestion
	types: ('allowed' | 'inconsistent')[];
```
I believe it is redundant to prefix it with the function name, this should be clear enough.

Reply:
Agreed. 👍 Updated in https://github.com/directus/directus/pull/24501/commits/360946f26a7db23083e0b61fa0fa569579732f27